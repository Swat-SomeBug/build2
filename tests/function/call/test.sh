#!/bin/sh

b -q | diff -u test.out -
