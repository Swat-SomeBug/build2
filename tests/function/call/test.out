identity()
identity(a)
identity(a b c)
identity(sub/dir{x} sub/dir{y} sub/dir{z})
FOOBAR
FOOBAR
FOOBAR
